README.md

```diff
@@ -40,8 +39,10 @@ Here's how to use it in a playbook:
 - hosts: all
   become: yes
   become_method: sudo
-  roles:
-    - chrony
+  tasks:
+    - name: Install and configure chrony
+      ansible.builtin.include_role:
+        name: chrony
```

Comment:
Also maybe mention in the example that it's being configured for use on AWS specifically.

```suggestion
        name: chrony_aws
```

Reply:
See https://github.com/cisagov/ansible-role-chrony-aws/pull/32/commits/df5a5c4d6e3bac4d8387669f89bda038e9065d7d 👍